Features/Context/PlatformUI.php

```diff
@@ -0,0 +1,423 @@
+        {
+            $jsCode = JsHelper::getHelperJs() . $jsCode;
+        }
+        $jsCode = $this->driver->wrappedFunction( $jsCode );
+
+        return $this->getSession()->evaluateScript( $jsCode );
+    }
+
+    /**
+     * Execute javascript code and return result.
+     *
+     * @param   string  $jsCode
+     * @return  mixed
+     */
+    protected function execJavascript( $jsCode, $addJsHelper = true )
+    {
+        if ( $addJsHelper )
+        {
+            $jsCode = JsHelper::getHelperJs() . $jsCode;
+        }
+
+        $jsCode = $this->driver->wrappedFunction( $jsCode );
+
+        return $this->getSession()->executeScript( $jsCode );
+    }
+
+    /**
+     * Waits for Javascript to finnish by running a empty Javascript
+     * (In Sahi it's possible to have the same result by running an empty javascript only)
+     */
+    protected function waitForLastJs()
+    {
+        //Needs to be here
+        $this->execJavascript( '' );
+
+        $jsCode = "return BDD.isSomethingLoading();";
+
+        while ( $this->evalJavascript( $jsCode ) )
+        {
+            usleep( 100 * 1000 ); // 100ms
+        }
+    }
+
+    /**
+     * Waits for Javascript to finnish by running a empty Javascript
+     * (In Sahi it's possible to have the same result by running an empty javascript only)
+     */
+    protected function activateJsErrorHandler()
+    {
+        //Needs to be here
+        //$this->execJavascript( '' );
+        $jsCode = "return BDD.errorHandlerActivate();";
+        $this->execJavascript( $jsCode );
+    }
+
+    /**
+     * @Given I create a content of content type :type with:
+     */
+    public function iCreateContentType( $type, TableNode $fields )
+    {
+        $this->clickNavigationZone( "Platform" );
+        $this->waitForJs();
+        $this->iClickAtLink( "Content structure" );
+        $this->waitForJs();
+        $this->clickActionBar( "Create a content" );
+        $this->waitForJs();
+        $this->clickContentType( $type );
+        $this->waitForJs();
+        foreach ( $fields as $fieldArray )
+        {
+            $keys = array_keys( $fieldArray );
+            for ( $i = 0; $i < count( $keys ); $i++ )
+            {
+                $this->fillFieldWithValue( $keys[$i], $fieldArray[$keys[$i]] );
+            }
+        }
+    }
+
+    /**
+     * @Then I see Content :contentName of type :contentType
+     */
+    public function contentExists( $contentName, $contentType )
+    {
+        $contentId = $this->getLocationId();
+        $content = $this->getContentManager()->loadContentWithLocationId( $contentId );
+        $contentInfo = $content->contentInfo;
+        $contentTypeName = $this->getContentManager()->getContentType( $content );
+        Assertion::assertEquals( $contentName, $contentInfo->name, "Content has wrong name" );
+        Assertion::assertEquals( $contentType, $contentTypeName, "Content has wrong type" );
+    }
+
+    /**
+     * @Given I fill in :field subform with:
```

Comment:
what is a *subform* ? Most likely, the original sentence should be written differently